database.sql

```diff
@@ -655,6 +655,8 @@ CREATE TABLE IF NOT EXISTS `notify` (
 	`seen` tinyint(1) NOT NULL DEFAULT 0,
 	`verb` varchar(255) NOT NULL DEFAULT '',
 	`otype` varchar(16) NOT NULL DEFAULT '',
+	`name_cache` tinytext,
```

Comment:
You create 2 new rows in the table notify. Please add the document about these rows here: https://github.com/friendica/friendica/blob/develop/doc/database/db_notify.md

@annando 
I totally forgot to check the db docu of your latest canges to the database. Can you also please add the needed docu of  your latest db changes?